src/DependencyInjection/AbstractSonataAdminExtension.php

```diff
@@ -53,6 +53,10 @@ protected function fixTemplatesConfiguration(
                         'choice' => '@SonataAdmin/CRUD/list_choice.html.twig',
                         'url' => '@SonataAdmin/CRUD/list_url.html.twig',
                         'html' => '@SonataAdmin/CRUD/list_html.html.twig',
+                        'many_to_many' => '@SonataAdmin/CRUD/Association/list_many_to_many.html.twig',
+                        'many_to_one' => '@SonataAdmin/CRUD/Association/list_many_to_one.html.twig',
+                        'one_to_many' => '@SonataAdmin/CRUD/Association/list_one_to_many.html.twig',
+                        'one_to_one' => '@SonataAdmin/CRUD/Association/list_one_to_one.html.twig',
```

Comment:
```php
if (isset($bundles['SonataDoctrineORMAdminBundle'])) {
    $defaultConfig['templates']['types']['list'] = array_merge($defaultConfig['templates']['types']['list'], [
        'many_to_many' => '@SonataAdmin/CRUD/Association/list_many_to_many.html.twig',
        'many_to_one' => '@SonataAdmin/CRUD/Association/list_many_to_one.html.twig',
        'one_to_many' => '@SonataAdmin/CRUD/Association/list_one_to_many.html.twig',
        'one_to_one' => '@SonataAdmin/CRUD/Association/list_one_to_one.html.twig',
    ]);	            
}

```

Reply:
I think this templates should not be add without storage bundle.

Reply:
No, these template are not SonataDoctrineORMAdminBundle-related.

Some time ago, these template was in SonataDoctrineORMAdminBundle, in SonataMonfoDBAdminBundle, ... but since it was the same template they were moved in this bundle.

As you can see, the path is `'@SonataAdmin/...`, they are always available, so should be added in this list.

Because it wasn't added in the list, others bundle can access to them is the correct way, and added some hack.

Reply:
@VincentLanglet thanks